.github/workflows/c-coverage.yml

```
@@ -0,0 +1,56 @@
+name: C coverage
+
+on:
+  schedule:
+    # Run this daily at 2400 UTC
+    - cron: '0 0 * * *'
+
+permissions:
+  contents: read
+
+jobs:
+  c-coverage:
+    name: 'C-level coverage (using llvm-cov)'
+    runs-on: ubuntu-20.04
+    env:
+      OPENSSL_VER: 1.1.1n
+      PYTHONSTRICTEXTENSIONBUILD: 1
+    steps:
+    - name: Install Dependencies
+      run: |
+        sudo ./.github/workflows/posix-deps-apt.sh
+    - name: Configure OpenSSL env vars
+      run: |
+        echo "MULTISSL_DIR=${GITHUB_WORKSPACE}/multissl" >> $GITHUB_ENV
+        echo "OPENSSL_DIR=${GITHUB_WORKSPACE}/multissl/openssl/${OPENSSL_VER}" >> $GITHUB_ENV
+        echo "LD_LIBRARY_PATH=${GITHUB_WORKSPACE}/multissl/openssl/${OPENSSL_VER}/lib" >> $GITHUB_ENV
+    - name: 'Restore OpenSSL build'
```

The other names are unquoted, so these seem unnecessary:
```suggestion
    - name: Restore OpenSSL build
```